src/ratingservice/05_app_engine.tf

```diff
@@ -0,0 +1,38 @@
+resource "google_app_engine_standard_app_version" "ratingservice_v1" {
+  version_id = "v1"
+  service    = "ratingservice"
+  runtime    = "python38"
+
+  entrypoint {
+    shell = "gunicorn -b :8080 main:app"
+  }
+
+  deployment {
+    zip {
+      # Need to zip source code and upload to GCS
+      source_url = "https://storage.googleapis.com/${var.bucket_name}/ratingservice_code.zip"
```

Comment:
Can you explain this a bit more? Is it automated?

Reply:
Yeah, we can keep the zip file in source code or zip them in install.sh. And then we need to upload the zip file to GCS.

Reply:
Is that automation coming later? I don't see it here

Reply:
This is not in scope now since the service will only be integrated afterwards. I won't modify the install.sh for now.

Reply:
I added it now.